internal/kafka/command_link.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 const (
-	configFileFlagName = "config-file"
-	dryrunFlagName     = "dry-run"
+	configFileFlagName           = "config-file"
+	dryrunFlagName               = "dry-run"
```

Comment:
None of our other commands define these as constants because we just get a runtime error if there's a misspelling somewhere. If you want to keep them, let's just make sure the formatting is the same.
```suggestion
	dryRunFlagName               = "dry-run"
```